config/initializers/devise.rb

```diff
@@ -102,11 +102,11 @@
   # access will be blocked just in the third day. Default is 0.days, meaning
   # the user cannot access the website without confirming his account.
 
-  # Disallow unconfirmed access
-  # Note: in spring of 2022 we are gradually requiring email confirmation by
-  # allowing existing users some time to sign in and manually send themselves
-  # a confirmation email. This is managed in User#active_for_authentication?
-  config.allow_unconfirmed_access_for = 0
+  # Allow unconfirmed access
+  # Note: in spring of 2022 we encouraging email confirmation and also required
+  # new user to have completed email confirmation before logging in. In spring
+  # 2024 we began once again allowing users to login before confirming their email.
+  config.allow_unconfirmed_access_for = 20.years
```

Comment:
"we encouraging" is a typo. 20 years is basically forever, not "a few years". Is that really the intent?